# Motor commands message
float32[] p_w
float32[] q_wb
bool tracking_valid
